types/jsdom/jsdom-tests.ts

```diff
@@ -1,9 +1,12 @@
 import "./test/core";
 import jsdom = require("jsdom");
+// tslint:disable-next-line:multiple-import-in-one-file test named export
+import { JSDOM } from "jsdom";
 
-const dom = new jsdom.JSDOM();
+const dom = new jsdom.JSDOM() || new JSDOM();
 const domWindow = dom.window; // $ExpectType DOMWindow
-
+dom.virtualConsole; // $ExpectType VirtualConsole
+dom.cookieJar; // $ExpectType CookieJar
```

Comment:
**Nit:** I prefer line breaks between grouped statements to match the existing formatting:
```suggestion

dom.virtualConsole; // $ExpectType VirtualConsole
dom.cookieJar; // $ExpectType CookieJar

```